pulpcore/cli/ansible/distribution.py

```diff
@@ -67,9 +67,11 @@ def create(
         body["repository_version"] = f'{repo["versions_href"]}{version}/'
     elif repository:
         body["repository"] = repo["pulp_href"]
+
     result = distribution_ctx.create(body=body)
-    distribution = distribution_ctx.show(result["created_resources"][0])
-    pulp_ctx.output_result(distribution)
+    if not pulp_ctx.background_tasks:
```

Comment:
I've seen, that we don't output anything from `update` calls elsewhere.
Which is in line with other unix shell commands like `mv`, `cp` and even `rm`.

Reply:
You commented on the creation method which does show the output for other commands, but this one is the only one that is an async task.  As for the other update commands, if wait is on you might as well show the result of the task.